homeworks/Makc.Sas_MakcSas/homework_6/homework_6_1.js

```diff
@@ -0,0 +1,33 @@
+const user = {};
+const phone = {};
+let fullName;
+let phoneNumber;
+Object.defineProperties(user, {
+    fullName: {
+        set: (name) => {
+            fullName = name.trim().toLowerCase().replace(/\b\w/g, char => char.toUpperCase());
+        },
+        get() {
+            return fullName;
+        },
+    },
+});
+Object.defineProperties(phone, {
+    phoneNumber: {
+        set: (numPhone) => {
+            const plusSymbol = /^\D*\+/g;
```

Comment:
RegExp is not needed to check if first letter is `+`, you could use `charAt()` for example